docs/content.go

```diff
@@ -363,9 +367,11 @@ Enterprise Performance Computing (EPC)`
 	CapabilityDropUse   string = `drop [drop options...] <capabilities>`
 	CapabilityDropShort string = `Remove capabilities from a user or group (requires root)`
 	CapabilityDropLong  string = `
-  Remove Linux capabilities from an user/group. NOTE: This command requires root to run.
+  Remove Linux capabilities from an user/group. NOTE: This command requires root 
```

Comment:
```suggestion
  Remove Linux capabilities from a user/group. NOTE: This command requires root 
```
Even though 'user' starts with a vowel, this 'u' is actually a glide/semivowel, meaning it is treated as a consonant.